kernel/aster-nix/src/fs/fs_resolver.rs

```diff
@@ -37,33 +37,33 @@ impl Default for FsResolver {
 impl FsResolver {
     pub fn new() -> Self {
         Self {
-            root: root_mount().root_dentry().clone(),
-            cwd: root_mount().root_dentry().clone(),
+            root: DentryMnt::new(root_mount().clone(), root_mount().root_dentry().clone()),
+            cwd: DentryMnt::new(root_mount().clone(), root_mount().root_dentry().clone()),
         }
     }
 
     /// Get the root directory
-    pub fn root(&self) -> &Arc<Dentry> {
+    pub fn root(&self) -> &Arc<DentryMnt> {
         &self.root
     }
 
     /// Get the current working directory
-    pub fn cwd(&self) -> &Arc<Dentry> {
+    pub fn cwd(&self) -> &Arc<DentryMnt> {
         &self.cwd
     }
 
     /// Set the current working directory.
-    pub fn set_cwd(&mut self, dentry: Arc<Dentry>) {
-        self.cwd = dentry;
+    pub fn set_cwd(&mut self, dentrymnt: Arc<DentryMnt>) {
+        self.cwd = dentrymnt;
     }
 
     /// Set the root directory
-    pub fn set_root(&mut self, dentry: Arc<Dentry>) {
-        self.root = dentry;
+    pub fn set_root(&mut self, dentrymnt: Arc<DentryMnt>) {
+        self.root = dentrymnt;
     }
 
     /// Open or create a file inode handler.
-    pub fn open(&self, path: &FsPath, flags: u32, mode: u16) -> Result<InodeHandle> {
+    pub fn open(&self, pathname: &FsPath, flags: u32, mode: u16) -> Result<InodeHandle> {
```

Comment:
If `Path` is named to `DentryMnt`, then renaming `path` to `pathname` would not be necessary, right?

Reply:
This is an issue I want to mention. In some places, the path name of the file is named path and in some places it is named pathname. In order to facilitate understanding, I think it may need to be unified, such as in
`kernel/aster-nix/src/syscall/chmod.rs` and `kernel/aster-nix/src/syscall/chdir.rs`.